dev/IconSource/IconSource.idl

```diff
@@ -5,6 +5,11 @@
 [webhosthidden]
 unsealed runtimeclass IconSource : Windows.UI.Xaml.DependencyObject
```

Comment:
>unsealed [](start = 0, length = 8)

since this is unsealed, does there need to be an overridable method like CreateIconElementOverride that externally derived classes can implement? (This may have been discussed in API review and I missed it)

Reply:
Since IconSource doesn't have a constructor it isn't actually derivable.